src/main/java/nl/novi/techiteasy1121/controllers/CIModuleController.java

```diff
@@ -0,0 +1,58 @@
+package nl.novi.techiteasy1121.controllers;
+
+
+import nl.novi.techiteasy1121.dtos.CIModuleDto;
+import nl.novi.techiteasy1121.services.CIModuleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+// Dit is de Controller klasse van de CIModule entiteit en heeft vergelijkbare methodes als de TelevisionController
+@RestController
+public class CIModuleController {
+    private final CIModuleService ciModuleService;
+
+    public CIModuleController(CIModuleService ciModuleService){
+        this.ciModuleService = ciModuleService;
```

Comment:
Het is goed dat je de Dependency Injection via de constructor hebt geïmplementeerd. Dit is een best practice en zorgt voor betere testbaarheid van je code.